tests/reg_tests/test_DVConstraints.py

```diff
@@ -719,6 +732,17 @@ def test_projectedArea_box(self, train=False, refDeriv=False):
                 atol=1e-7,
             )
 
+    def test_projectedArea_box_funcSens(self, train=False, refDeriv=False):
+        np.random.seed(37)
```

Comment:
Great test to add, thanks. Same minor comment about the random number generator as previously.